src/Application/Actions/GetPasswordResetToken.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace BigGive\Identity\Application\Actions;
+
+use BigGive\Identity\Repository\PasswordResetTokenRepository;
+use BigGive\Identity\Repository\PersonRepository;
+use Laminas\Diactoros\Response\JsonResponse;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Log\LoggerInterface;
+use Slim\Exception\HttpNotFoundException;
+use Symfony\Component\Uid\Uuid;
+
+class GetPasswordResetToken extends Action
+{
+    public function __construct(
+        LoggerInterface $logger,
+        private readonly PasswordResetTokenRepository $tokenRepository,
+    ) {
+        parent::__construct($logger);
+    }
+
+    protected function action(Request $request, array $args): Response
+    {
+        $uuid = Uuid::fromBase58((string) ($args['base58'] ?? throw new HttpNotFoundException($request)));
```

Comment:
I'd call it `token` for consistency:

```suggestion
        $uuid = Uuid::fromBase58((string) ($args['token'] ?? throw new HttpNotFoundException($request)));
```

Or:
```suggestion
        $uuid = Uuid::fromBase58((string) ($args['base58Token'] ?? throw new HttpNotFoundException($request)));
```